docs/docs/sample-scripts/spo/copy-listitems-sharepointlist.md

```diff
@@ -0,0 +1,35 @@
+# Copy list items between SharePoint lists
+
+Author: [SekThang](https://twitter.com/SekThang), Inspired by [Ruud](https://lazyadmin.nl/it/copy-sharepoint-list-items-to-another-list-with-powershell-and-pnp/)
+
+The cli script helps you to copy list items from one list to another list.
+I have written script logics to migrate list items from one site collection to another site collection
+
+- Prerequisites: List and metadata should be created in the destination site collection site as it's in the source site collection
+- Kindly update columns in the line number 24 as your needs
+
+```powershell tab="PowerShell"
+$SourceSite = Read-Host -Prompt 'Source site Url'
+$DestinationSite = Read-Host -Prompt 'Desitnation site Url'
+$SourceList = Read-Host -Prompt 'Source list name'
+$DestinationList = Read-Host -Prompt 'Destination list name'
+
+$listItems = m365 spo listitem list --title $SourceList --webUrl $SourceSite --output json | ConvertFrom-Json
+Write-Host 'Total count in the source list is'-> -fore Green $listItems.Count
+$count = 0
+$reportLocation = Get-Location 
+foreach($item in $listItems)
+{
+	  $count++
+		m365 spo listitem add --listTitle $DestinationList --webUrl $DestinationSite --Title $item.Title --Firstname $item.Firstname --Lastname $item.Lastname
```

Comment:
When copying list items with multiple lines, I see an extra `\` in the copied item.

Copied:
![image](https://user-images.githubusercontent.com/11164679/141963095-1d771986-ac6c-4535-b080-a7d3aec31e0c.png)

Original:
![image](https://user-images.githubusercontent.com/11164679/141963133-086cd992-78da-45b3-ab14-4e346a65d5f1.png)

Could you please have a look at it?